clients/cmd/promtail/main.go

```diff
@@ -78,11 +78,11 @@ func main() {
 	}
 
 	// Init the logger which will honor the log level set in cfg.Server
-	if reflect.DeepEqual(&config.ServerConfig.Config.LogLevel, &logging.Level{}) {
+	if reflect.DeepEqual(&config.Config.ServerConfig.Config.LogLevel, &logging.Level{}) {
```

Comment:
My linter required this :shrug:

Reply:
Just out of interest what was the message?